packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -149,6 +149,8 @@ export interface CallAdapterCallManagement {
     createStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void | CreateVideoStreamViewResult>;
     disposeStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void>;
     // @beta
+    getEnvironmentInfo(): Promise<EnvironmentInfo>;
```

Comment:
Is there a reason to only expose `getEnvironmentInfo` and not the `DebugInfo` interface?

Reply:
from my understanding to use the `DebugInfo` interface you need to use the `feature()` function on the call client which we proxied to get this API. If Contoso wanted the `DebugInfo` interface they could get it off the `StatefulClient` and then create the adapter from there. everything else on the interface didn't seem relevant to the feature so I didn't add it. do you think we should?